trio/_core/_tests/test_guest_mode.py

```diff
@@ -301,7 +301,7 @@ async def sit_in_wait_all_tasks_blocked(watb_cscope):
                 # wait_all_tasks_blocked should *not* return normally, but
                 # only by cancellation.
                 await trio.testing.wait_all_tasks_blocked(cushion=9999)
-                assert False  # pragma: no cover
+                raise AssertionError()  # pragma: no cover
```

Comment:
same - though here we have a comment we could copy text from.

Reply:
Added reason text in a5afed7